flowzone.yml

```
@@ -3120,15 +3120,30 @@ jobs:
 
       - name: Generate shared outputs
         id: shared
+        env:
+          SECRETS_CONTEXT: ${{ toJson(secrets) }}
+          VARS_CONTEXT: ${{ toJson(vars) }}
         run: |
-          stack="$(echo '${{ inputs.cloudformation_templates }}' | jq -r '.stacks[] | select(.name=="${{ matrix.stack }}")')"
+          set -a
+
+          cleanup() { rm -f .env; }
+          trap cleanup EXIT
```

```suggestion
          trap 'rm -f .env' EXIT
```
Combine these two lines into one.